frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -140,8 +202,8 @@ export class ImportWizard extends Wizard {
     await asyncForEach(importedDisks, async (disk) => {
       await clickKebabAction(disk.name, 'Edit');
       await waitForNoLoaders();
-      // Change name to comply with k8s
-      await diskDialog.fillName(disk.name.replace(/\s/g, '').toLowerCase());
+      // Change name to comply with k8s is not required now, name is correct from scratch
+      // await diskDialog.fillName(disk.name.replace(/\s/g, '').toLowerCase());
       // This if is required to workaround bug 1814611.
       // NFS is not supported for conversion PV and HPP should be used instead.
       if (disk.name === 'v2v-conversion-temp' && STORAGE_CLASS === 'nfs') {
```

Comment:
Similar with this method, we should rename it to something like `configureConversionDisk` maybe?
Additionally, the condition would be better to be `STORAGE_CLASS !== 'hostpath-provisioner'`

Reply:
We have not only hostpath-provisioner or NFS, we have also CEPH and local storage. 
So regarding that if - let me disagree with you and remain it as is.

Reply:
Ah so it works fine with the others? okay then

Reply:
Yep. Anyway, this bug should be fixed soon and I will remove this checking.